src/main/java/com/vandendaelen/k9/entities/EntityK9.java

```diff
@@ -106,8 +106,8 @@ public boolean processInteract(EntityPlayer player, EnumHand hand) {
 
             if (player.getHeldItem(hand).getItem() instanceof ItemK9Remote){
                 ItemK9Remote remote = (ItemK9Remote) player.getHeldItem(hand).getItem();
-                remote.setK9ID(player.getHeldItem(hand), this.getUniqueID());
-                K9.logger.debug(remote.getK9ID(player.getHeldItem(hand)));
+                ItemK9Remote.setK9ID(player.getHeldItem(hand), this.getUniqueID());
+                K9.logger.debug(ItemK9Remote.getK9ID(player.getHeldItem(hand)));
```

Comment:
I don't get this change, why use the Class instead of the Object ?

Reply:
I blame my IDE for that one,,,,,